### ISSUE TITLE
Explore for copy-dbadbtabledata

### ISSUE THREAD

**Issue by potatoqualitee** [sqlcollaborative/dbatools#5767]
https://stackoverflow.com/questions/11658143/how-to-set-smo-scriptingoptions-to-guarantee-exact-copy-of-table

**Comment by potatoqualitee** [comment]
I'm not so sure -- ppl seem happy with copy-dbadbtabledata. Maybe this would be copy-dbadbtable instead?

**Comment by mazorg** [comment]
Actually We would appreciate Copy-DbaDbTableData had options to override the defaultFGScriptingOption

**Comment by andreasjordan** [comment]
@claude please give me some advice on this.

**Comment by andreasjordan** [comment]
@claude yes, please implement Option 1 (add -ScriptingOptionsObject to Copy-DbaDbTableData).
